ihatemoney/models.py

```diff
@@ -320,6 +322,42 @@ def switch_currency(self, new_currency):
         db.session.add(self)
         db.session.commit()
 
+    def import_bills(self, bills: list):
+        """Import bills from a list of dictionaries"""
+        # Add members not already in the project
+        members_project = [str(m) for m in self.members]
+        members_new = [
+            m for m in get_members(bills) if str(m[0]) not in members_project
+        ]
+        for m in members_new:
+            Person(name=m[0], project=self, weight=m[1])
+        db.session.commit()
+
+        # Import bills not already in the project
+        bills_project = self.get_pretty_bills()
```

Comment:
I'm not sure I follow here: this is named `bills_project` but it seems to contain bills instead, right? If that's the case, we should probably rename the variable here.

Reply:
I think I understand. It seems to be the bills of the existing project. If this is the case, we could name it `project_bills` :-)

Reply:
Ok !